src/toast/__tests__/Toast.test.jsx

```diff
@@ -44,7 +44,7 @@ describe('Toast', () => {
   });
 
   afterAll(() => {
-    global.document.getElementById('toast').remove();
+    global.document.getElementById('toast-root').remove();
   });
```

Comment:
This test did not make sense either. There is no element with id 'toast'. Previous version of JSDOM may have worked more like jQuery and returned callable functions when query does not return anything. But now calling remove() threw and error.

Also there is only one test, so unsuccessful removal in afterAll() have no effect to following tests.